charts/values.global.yaml

```diff
@@ -16,29 +16,31 @@ commonLabels: false
 namespace:
   public: gmp-public
   system: gmp-system
-version: 0.13.0
+version: 0.13.1
 images:
+  # NOTE: All tags have to be quoted otherwise they will be treated as a number.
   bash:
     image: gke.gcr.io/gke-distroless/bash
-    tag: "gke_distroless_20240607.00_p0" # NOTE: Has to be quoted otherwise it will be treated as a number.
+    tag: "gke_distroless_20240907.00_p0"
   alertmanager:
```

Comment:
is it possible to update the alertmanager image at this stage of the release or no?

Reply:
Not yet, we will do once 0.14.0 is out with manually bumped AM new image.

Reply:
However we can follow what @pintohutch suggested and did in 0.13 branch - change things on release/0.14 to what we expect to release once prod images are there, then cherry pick to main once done. Will try that.